projects/Bactteria_Dataset/README.md

```diff
@@ -0,0 +1,114 @@
+# Bactteria detection with darkfield microscopy Dataset
+
+This project support **`Bactteria detection with darkfield microscopy Dataset`**, and the dataset can be download from [here](https://tianchi.aliyun.com/dataset/94411).
+
+## Usage
+
+<!-- For a typical model, this section should contain the commands for training and testing. You are also suggested to dump your environment specification to env.yml by `conda env export > env.yml`. -->
+
+### Prerequisites
+
+- Python 3.8
+- PyTorch 1.10.0
+- [MIM](https://github.com/open-mmlab/mim) v0.3.4
+- [MMcv](https://github.com/open-mmlab/mmcv) v2.0.0rc3
+- [MMEngine](https://github.com/open-mmlab/mmengine) v0.1.0 or higher
+- [MMClassification](https://github.com/open-mmlab/mmclassification) v1.0.0rc4
+- [MMDetection](https://github.com/open-mmlab/mmdetection) v3.0.0rc5
+- [MMSegmentation](https://github.com/open-mmlab/mmsegmentation) v1.0.0rc3
+
+### Dataset preparing
+
+Preparing `Bactteria detection with darkfield microscopy Dataset` dataset in following format as below.
```

Comment:
It is doubtful whether this directory is the result of processing before or after using the script. As the `random_split.py` is in the below.